dbsnap_verify/__init__.py

```diff
@@ -0,0 +1,176 @@
+from rds_funcs import (
+    get_latest_snapshot,
+    get_database_description,
+    restore_from_latest_snapshot,
+    modify_db_instance_for_verify,
+    dbsnap_verify_db_id,
+    destroy_database,
+    destroy_database_subnet_group,
+    rds_event_messages,
+)
+
+from state_doc import (
+    current_state,
+    transition_state,
+    get_or_create_state_doc,
+)
+
+from time_funcs import (
+    tomorrow_timestamp,
+    now_datetime,
+    timestamp_to_datetime,
+    datetime_to_date_str,
+    three_days_prior,
+)
+
+import boto3
+
+from sys import stdout
+import logging
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+logger.addHandler(logging.StreamHandler(stdout))
+
+
+def wait(state_doc, rds_session):
```

Comment:
Can we get some docstrings on these methods?  The names themselves aren't very friendly, unless you're intimately knowledgeable about the design/architecture.

Reply:
Great point, I'll add this in.